cscs-checks/apps/gromacs/gromacs_check.py

```diff
@@ -119,6 +206,11 @@ def setup_run(self):
         if self.current_partition.fullname in ('daint:gpu', 'dom:gpu'):
             arch = 'sm_60'
 
+        try:
+            found = self.allref[self.num_nodes][arch][self.bench_name]
+        except KeyError:
+            self.skip("Test configuration is not supported")
```

Comment:
```suggestion
            self.skip('Test configuration is not supported')
```

Also a bit more info on the unsupported configuration would help.